core/src/main/java/org/apache/calcite/sql/validate/SqlValidator.java

```diff
@@ -912,15 +912,39 @@ interface Config {
      */
     Config withLenientOperatorLookup(boolean lenient);
 
-    /** Returns whether the validator allows measures to be used without the
-     * AGGREGATE function. Default is true. */
-    @Value.Default default boolean nakedMeasures() {
+    /** Returns whether the validator allows measures to be used without
+     * AGGREGATE function outside an aggregating query. Default is true.
+     */
```

Comment:
move `*/` to end of previous line, to be consistent with formatting in this file

Reply:
There are 96 `/**` doc comments in this source file, 16 of which don't have `*/` on a separate line at the end, and of those, 10 are single-line doc comments. Only 6 doc comments are multi-line with `*/` at the end of the last line, rather than on a line on its own. I changed the style of this comment when editing to bring it into greater consistency with the majority of the rest of the file.

I didn't do an exahustive analysis, but a quick scan of other source files in this directory shows they generally have doc comments with `*/` at the end on a line of its own.